example/typescript/toot.ts

```diff
@@ -1,12 +1,13 @@
 import * as readline from 'readline'
-import Mastodon from '../../src/mastodon'
+import Mastodon from 'megalodon'
+import { Status } from 'megalodon'
```

Comment:
__[TSLint]__

Multiple imports from 'megalodon' can be combined into one. (no-duplicate-imports)


<!-- Comment by [Sider](https://sider.review) -->